daemon/create.go

```diff
@@ -34,6 +34,9 @@ func (daemon *Daemon) ContainerCreate(name string, config *runconfig.Config, hos
 	if hostConfig.Memory == 0 && hostConfig.MemorySwap > 0 {
 		return "", warnings, fmt.Errorf("You should always set the Memory limit when using Memoryswap limit, see usage.\n")
 	}
+	if hostConfig.BlkioWeight > 0 && (hostConfig.BlkioWeight < 10 || hostConfig.BlkioWeight > 1000) {
+		return "", warnings, fmt.Errorf("Range of blkio weight is from 10 to 1000.\n")
```

Comment:
No \n at the end of the error message please (leave it to the printing code, if any)!